notie.js

```diff
@@ -100,6 +96,25 @@ var notie = function(){
     alert_inner.style.verticalAlign = 'middle';
     alert_outer.appendChild(alert_inner);
 
+    if (typeof Element.prototype.addEventListener === 'undefined') {
+        Element.prototype.addEventListener = function (e, callback) {
```

Comment:
maybe doing this instead of replicating the code:
```js
Element.prototype.addEventListener = Window.prototype.addEventListener = function (e, callback) {
    // ...
}
```